packages/helium-cli/emissions/iot.json

```diff
@@ -1,14 +1,14 @@
 [
   {
-    "startUnixTime": 1679064129,
-    "emissionsPerEpoch": "165616438356164"
+    "startTime": "2023-04-18T00:00:00Z",
+    "emissionsPerEpoch": "165,616,438.356164"
   },
   {
-    "startUnixTime": 1690848000,
-    "emissionsPerEpoch": "82808219178082"
+    "startTime": "2023-08-01T00:00:00Z",
+    "emissionsPerEpoch": "82,581,967.213115"
```

Comment:
`82,581,967.213115 * (100/93) * 366 = 32.5B` annualized for the second year including the leap day